src/components/modals/BottomModal/index.tsx

```diff
@@ -47,7 +51,7 @@ const BottomModal = (props: BottomModalProps): JSX.Element => {
       easing={Easing.ease}
     >
       <View style={tailwind('h-full')}>
-        <StatusBar translucent />
+        <StatusBar style={statusBarStyle} translucent={Platform.OS === 'android'} />
```

Comment:
Perhaps you can extract Platform.OS === 'android' to a constant like const shouldBeTranslucent = Platform.OS === 'android'.